src/components/ConversationSettings/LockingSettings.vue

```diff
@@ -23,6 +23,7 @@
 	<div class="app-settings-subsection">
 		<div id="moderation_settings_lock_conversation_hint" class="app-settings-section__hint">
 			{{ t('spreed', 'Locking the conversation prevents anyone to post messages or start calls.') }}
+			{{ t('spreed', 'This will also terminate ongoing calls.') }}
```

Comment:
I would move it to a new div/p tag and use `v-if="hasCall"`

```suggestion
			{{ t('spreed', 'This will also terminate the ongoing call.') }}
```